driver/src/transport.rs

```diff
@@ -63,7 +63,7 @@ impl HttpTransportInner {
             .post_raw_json_async(&self.url, request, label)
             .await
             .map_err(|err| {
-                warn!("[id:{}] returned an error: '{}'", id, err.to_string());
+                warn!("[id:{}] returned an error: '{:?}'", id, err);
```

Comment:
nit: This error should get propagated, and printed in full at a higher level, I think here it should be ok to stay with `Display` instead of `Debug`.

Reply:
With that reasoning we don't need to print it at all but I'll change it back.